src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,59 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        try {
```

Comment:
it's recommended to have try block as little as possible. It should only contain code that generates exception. Let's move forming of report outside of the try block